packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -653,25 +653,25 @@ class FilterButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final colorScheme = Theme.of(context).colorScheme;
+    final theme = Theme.of(context);
     return RoundedOutlinedBorder(
       child: SizedBox(
-        height: defaultButtonHeight,
+        height: theme.buttonTheme.height,
```

Comment:
Please don't use ButtonTheme, it's obsolete. The component defaults are documented but are not readily available via the widgets or themes. The default height for TextButton is 64 per minimumSize in https://api.flutter.dev/flutter/material/TextButton/defaultStyleOf.html.

Reply:
removed all uses of buttonTheme